ocean_lib/ocean/mint.py

```diff
@@ -21,40 +21,36 @@
 
```

Comment:
Please remove `setup_enforce_typing_shim()` and its requisite import statement. They are not required because methods in this file are no longer executed before the static `ConfigProvider` object has been initialized. As a result, `isort:skip_file` and all instances of `# noqa: E402` should also be removed.

Reply:
Done :)